src/containerapp/azext_containerapp/_utils.py

```diff
@@ -103,15 +103,6 @@ def process_service(cmd, resource_list, service_name, arg_dict, subscription_id,
         raise ResourceNotFoundError("Service with the given name does not exist")
 
 
-def get_linker_client(cmd):
-    resource = cmd.cli_ctx.cloud.endpoints.active_directory_resource_id
-    profile = Profile(cli_ctx=cmd.cli_ctx)
-    credential, subscription_id, _ = profile.get_login_credentials(
-        subscription_id=get_subscription_id(cmd.cli_ctx), resource=resource)
-    linker_client = ServiceLinkerManagementClient(credential)
-    return linker_client
```

Comment:
This function is dup with [this](def get_linker_client(cmd):) and unused.
Due to in the azure-cli, it drops argument resource: https://github.com/Azure/azure-cli/pull/29631
We need to write it like this: https://github.com/Azure/azure-cli/pull/30799

Otherwise the CI will fail.